### ISSUE TITLE
"IAudioCaptureClient::setup failed"

### ISSUE THREAD

**Issue by ghost** [rdp/virtual-audio-capture-grabber-device#25]
Using ShareX, freshly installed with ffmpeg and recorder devices, and it spits out this error. 

`leaving aero onIAudioCaptureClient::setup failed`

I've had this error in the past and I can't remember how to fix it, I've searched endlessly and tried different ffmpeg versions as well. There's a couple open issues regarding this exact one on the ShareX github so thought maybe I'd post here instead.

If you wouldn't mind helping me figure this out I'd appreciate it.

**Comment by Dementei** [comment]
Gives me this error when trying to open it:
`IAudioClient::Initialize failed: hr = 0x88890008`
`Thread aborted before starting to loopback capture: hr = 0x88890008`

**Comment by rdp** [comment]
No idea unfortunately.
Googling it said something about "supported formats" hrmmm
https://social.msdn.microsoft.com/Forums/windowsdesktop/en-US/9acbda3f-1c9a-484e-a5aa-02b56b3b2616/isformatsupported-return-0x88890008-for-waveformatdolbyac3spdif-with-44100-sample-rate-in?forum=windowspro-audiodevelopment

**Comment by Dementei** [comment]
Mmmm, came across this https://github.com/coreyauger/AudioLoopbackFilter but unable to compile it, I can't install the SDK for it. Would that work with ShareX as a device?

**Comment by Dementei** [comment]
Is there a way to debug your Screen Capturer Recorder btw? It does the same thing and is unable to record with virtual audio capturer device, I'm guessing from same problem.

**Comment by rdp** [comment]
Right.

**Comment by rdp** [comment]
That one's about the same.  I wonder what the difference is here,
though...I assume you have headphones or speakers plugged in?  There are
other that may work "VAC" or what not
http://betterlogic.com/roger/2018/04/8972/

**Comment by Dementei** [comment]
After multiple reinstalls of the ShareX Recorder Devices it seems to work now, although I did update my audio drivers before this last reinstall, so that might have been it. Either way it's all good now, thanks.

**Comment by Dementei** [comment]
The error is back today, haven't even restarted my PC since it was working, wonder if there's a service or program that is hindering the device somehow.

**Comment by darkgreen** [comment]
I was able to resolve this by changing audio codec from AAC to MP3 ... now recording screen capture plus audio.

**Comment by darkgreen** [comment]
I was able to resolve this by changing audio codec from AAC to MP3 ... now recording screen capture plus audio.

**Comment by darkgreen** [comment]
I was able to resolve this by changing audio codec from AAC to MP3 ... now recording screen capture plus audio.

**Comment by darkgreen** [comment]
I was able to resolve this by changing audio codec from AAC to MP3 ... now recording screen capture plus audio.

**Comment by onlytanmoy** [comment]
Looking for some help capturing screen with audio using ShareX. Under 'screen recording options' in audio source I see Microphone and Internal AUX jack, no virtual audio capture. Using latest version of ShareX and downloaded FFMpeg.exe.

**Comment by rdp** [comment]
Hmm maybe open a new issue for that one (hint: try installing it
straight from my stuff, if that doesn't fail, complain to ShareX :)

**Comment by onlytanmoy** [comment]
> Hmm maybe open a new issue for that one (hint: try installing it straight from my stuff, if that doesn't fail, complain to ShareX :)
> […](#)
> On 9/13/20, onlytanmoy ***@***.***> wrote: Looking for some help capturing screen with audio using ShareX. Under 'screen recording options' in audio source I see Microphone and Internal AUX jack, no virtual audio capture. Using latest version of ShareX and downloaded FFMpeg.exe. -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#25 (comment)](https://github.com/rdp/virtual-audio-capture-grabber-device/issues/25#issuecomment-691712043)

Thanks for your response. Please point me to your stuff.

**Comment by zimirrr** [comment]
Had same issue, win10.
`leaving aero onIAudioCaptureClient::setup failed`

I've deleted all sound related programs in 'Add or remove programs'.
One of them was driver for my motherboard for Realtek that I installed manually. After uninstall finished it asked to reboot.

After reboot I've went to start -> Settings -> Update & security -> Check for updates and it updated my Realtek driver.

The error was gone.

**Comment by eru-vr** [comment]
I'm using voicemeeter banana and from time to time I use my bluetooth headphones. If I switch output to those bluetooth headphones in VM, that error appears and I can't record videos, but if I switch to my default speakers, it works fine.

**Comment by rdp** [comment]
I don't know if I can fix the issue?

**Comment by Sgale952** [comment]
I am using an Asus Strix audio card. I had the same error, but I was able to fix it by simply closing the Sonic Studio driver. Also, the Strix sound card doesn't show up in FFmpeg's device list for some reason
